Dockerfile-dev

```diff
@@ -67,4 +67,7 @@ RUN virtualenv --no-site-packages -p python2 /opt/certbot/venv && \
 # this might also help in debugging: you can "docker run --entrypoint
 # bash" and investigate, apply patches, etc.
 
+# installs git into the Docker image, as required by tox
+RUN apt-get update && apt-get install git -y
```

Comment:
nit: This should be done during the RUN command above that already does `apt-get update` and `apt-get install` so `apt-get update` is only done once and data such as repo lists can be cleaned up decreasing the size of the image.